src/frontend-pwa/src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+export type ToggleProps = {
+  accessibility: string;
+  onChange: (checked: boolean) => void;
+  defaultChecked: boolean;
+}
+
+export default function Toggle({
+  accessibility,
```

Comment:
make prop name more clear for use.
prop names should be meaningful enough that a dev won't have to go to your component file and figure it out
suggestion: accessibilityLabel, ariaLabel, etc